test/molecule-role/molecule/default/prepare.yml

```diff
@@ -125,6 +125,58 @@
         state: present
       when: ansible_distribution == "CentOS"
 
+- name: Prepare Network Connection Namespaces VM (pre)
+  hosts: agent_linux_connection_namespace_vm
+  vars_files:
+    - common_vars.yml
+  tasks:
+    - name: Install python for Ansible
+      raw: test -e /usr/bin/python || (apt -y update && apt install -y python-minimal python-zipstream)
+      become: true
+      changed_when: false
+    - name: Install python-pip
+      become: true
+      apt:
+        name: python-pip
+        state: present
+        update_cache: yes
+    - name: Installing multiple python packages in Ansible
+      pip:
+        name: boto,boto3
+    - name: Add Docker GPG APT key
+      become: true
+      apt_key: url=https://download.docker.com/linux/ubuntu/gpg
+    - name: Add Docker APT repository
+      become: true
+      apt_repository:
+        repo: deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable
+    - name: Install APT packages
+      become: true
+      apt:
+        name: [apt-transport-https, ca-certificates, curl, software-properties-common, docker-ce]
+        state: present
+        update_cache: yes
+      register: apt_res
+      retries: 5
+      delay: 5
+      until: apt_res is success
+    - name: Add user to docker group
+      become: true
+      user:
+        name: ubuntu
+        groups: docker
+        append: True
+    - name: Restart sshd to apply group change
+      shell: sleep 1; pkill -u {{ ansible_ssh_user }} sshd
+      async: 3
+      poll: 2
+    - name: Cleanup docker containers
+      shell: "docker container prune --force"
+    - name: Setup netcat 1 docker container
+      shell: "docker run -it -d --name netcat-1 ubuntu && docker exec -itd --user root netcat-1 sh -c 'apt-get update && apt-get install -y netcat && nc -lv -p 9091'"
```

Comment:
use `ncat --broker` instead of `nc` so you get correctly directions on connections